src/test/mod.rs

```diff
@@ -373,6 +373,11 @@ fn check_files(files: Vec<PathBuf>, opt_config: &Option<PathBuf>) -> (Vec<Format
     let mut reports = vec![];
 
     for file_name in files {
+        // this file name is used for module test
+        if file_name.ends_with("no_entry.rs") {
```

Comment:
If you don't mind, could you rebase against master and add `issue-3434/no_entry.rs` to `SKIP_FILE_WHITE_LIST` instead, please?

https://github.com/rust-lang/rustfmt/blob/393d7217ae990e05e036df4785ae5143da6aff2e/src/test/mod.rs#L21-L25